api/src/main/java/org/codice/ddf/admin/api/handler/report/ProbeReport.java

```diff
@@ -74,8 +74,18 @@ public ProbeReport probeResults(Map<String, Object> results) {
         return probeResults;
     }
 
+    public <T> T getProbeResult(String key) {
+        if(probeResults.get(key) != null) {
```

Comment:
I believe (and @tbatie can correct me if I'm wrong) that we need to do this in order to avoid trying to cast `null` to `T` upon return.